packages/features/webhooks/lib/sendPayload.ts

```diff
@@ -159,16 +191,21 @@ const sendPayload = async (
   const contentType =
     !template || jsonParse(template) ? "application/json" : "application/x-www-form-urlencoded";
 
-  data.description = data.description || data.additionalNotes;
+  if ("description" in data && "notes" in data) {
+    data.description = data.description || data.notes;
+  }
   data = addUTCOffset(data);
 
   let body;
-
   /* Zapier id is hardcoded in the DB, we send the raw data for this case  */
-  if (appId === "zapier") {
-    body = getZapierPayload({ ...data, createdAt });
+  if (appId === "zapier" && triggerEvent !== "OOO_CREATED") {
+    const zapierData = data as WithUTCOffsetType<
```

Comment:
Not a fan of doing this but this fix the root problem of using many types in WebhookDataType, that was one of the reasons of logic separation there. Or if you have another way to solve this (zod schema parsing) let me know.
@CarinaWolli